core/modules/views/includes/view.inc

```diff
@@ -2003,6 +2009,11 @@ class view {
       // property set to FALSE, signifying localization should not occur.
       if (empty($this->localization_plugin)) {
         $this->localization_plugin = views_get_plugin('localization', 'none');
+        // If it is still NULL, not even plugin "none" is available. Only
+        // needed for edge cases, like core update.
```

Comment:
"needed" still fits in the previous line, w/o exceeding the 80 char limit.

```suggestion
        // If it is still NULL, not even plugin "none" is available. Only needed
        // for edge cases, like core update.
```

Reply:
Problem: then it's 81 chars. :wink:

Reply:
No, it's not:

![Screen Shot 2020-09-30 at 5 32 00 pm](https://user-images.githubusercontent.com/2423362/94699435-fbabd780-0342-11eb-9ad7-ec13afbac2ed.png)

Reply:
I'm not sure how to deal with such passion regarding an integer (80). Better not argue...